error:
       … while calling the 'addErrorContext' builtin
         at /pwd/lang/eval-fail-addErrorContext-example.nix:6:7:
            5|     else
            6|       builtins.addErrorContext
             |       ^
            7|         "while counting down; n = ${toString n}"

       … while counting down; n = 10

       … while calling the 'addErrorContext' builtin
         at /pwd/lang/eval-fail-addErrorContext-example.nix:6:7:
            5|     else
            6|       builtins.addErrorContext
             |       ^
            7|         "while counting down; n = ${toString n}"

       … while counting down; n = 9

       … while counting down; n = 8

       … while counting down; n = 7

       … while counting down; n = 6

       … while counting down; n = 5

       … while counting down; n = 4

       … while counting down; n = 3

       … while counting down; n = 2

       … while counting down; n = 1

       (stack trace truncated; use '--show-trace' to show the full, detailed trace)

       error: kaboom
